catchpoint-mixin/targets.libsonnet

```diff
@@ -105,15 +106,235 @@ local utils = commonlib.utils {
     topMaxRedirectsTestName:
       prometheusQuery.new(
         '${' + vars.datasources.prometheus.name + '}',
-        'topk by (test_name) (2, max by (instance, test_name) (catchpoint_redirect_time{%(testNameSelector)s}))' % vars
+        'topk by (test_name, node_name) (2, max by (instance, test_name, node_name) (catchpoint_redirect_time{%(testNameSelector)s}))' % vars
       )
-      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(maxPanelLegend)),
 
     topErrorsByTestName:
       prometheusQuery.new(
         '${' + vars.datasources.prometheus.name + '}',
         'topk by (test_name) (2, max by (instance, test_name) (catchpoint_any_error{%(testNameSelector)s}))' % vars
       )
       + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+
+    // Web performance by test name dashboard
+    pageCompletionTime:
```

Comment:
This metric panel will need to include total load and document complete as separate timeseries.